modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb

```diff
@@ -0,0 +1,82 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = GoodRanking
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'            => 'Trixbox CE <= v2.8.0.4 Authenticated Remote Command Execution',
```

Comment:
You cannot have the characters < or = in the title. Please remove these characters as its currently making the Jenkins tests fail:

```
modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb - [INFO] No CVE references found. Please check before you land!
modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb - [ERROR] '<' is a bad character in module title.
modules/exploits/unix/webapp/trixbox_ce_auth_rce.rb - [ERROR] '=' is a bad character in module title.
```

Reply:
Confirmed changed.